dask_cloudprovider/providers/aws/ecs.py

```diff
@@ -73,6 +73,12 @@ class Task:
         Configurable timeout in seconds for finding the task IP from the
         cloudwatch logs.
 
+    name: str (optional)
+        Name for the task. Currently used for the --namecommand line argument to dask-worker.
+
+    use_private_ip: bool (optional)
+        Whether to use a private IP (if True) or public IP (if False). Default is False.
```

Comment:
This is still for fargate only, though, no? On ec2, this arg makes no difference.

Reply:
That's correct. After your comment and thinking about it more, I changed this to `fargate_use_public_ip` to clarify that it will only have an effect for Fargate and not ec2. (Sorry for the back-and-forth, but thanks for bearing with me!)